### PR TITLE
Implement and test optstatus for Gaussian

### PR DESCRIPTION
**Pull request by langner**

### CODE REVIEW

File: src/cclib/parser/gaussianparser.py [hunk 1]
```diff
@@ -213,14 +214,24 @@ def extract(self, inputfile, line):
 
             if not hasattr(self, 'optdone'):
                 self.optdone = []
-
             self.optdone.append(len(self.geovalues) - 1)
 
+            if not hasattr(self, 'optstatus'):
+                self.optstatus = []
+            assert len(self.optstatus) > 0
```
Comment (ATenderholt) [inline src/cclib/parser/gaussianparser.py]: Shouldn't this throw an assertion if line 220 is ever called (i.e. ```optstatus``` isn't created until now)? Same with line 232.

**Comment by ATenderholt** [comment]
```testScan.py``` also has checks for ```optdone```, so should be updated for ```optstatus```.

File: src/cclib/parser/gaussianparser.py (hunk 1 above)
Comment (langner) [inline src/cclib/parser/gaussianparser.py]: > Shouldn't this throw an assertion if line 220 is ever called (i.e. optstatus isn't created until now)? Same with line 232.

Quite right! Updated.

**Comment by langner** [comment]
> testScan.py also has checks for optdone, so should be updated for optstatus.

Done.

**Pull request merged**